src/pages/404.tsx

```diff
@@ -10,14 +10,14 @@ const StyledPage = styled(Page)`
   margin-top: 4rem;
 `
 
-const NotFoundPage = () => (
+const NotFoundPage: React.FC = () => (
```

Comment:
Since this is a Page component, we are not defining it as a `React.FC`. Lets keep the consistency across the other pages by only defining the props with `PageProps`.
```suggestion
const NotFoundPage = (props: PageProps) => {
```